packages/core/docs/performance/optimizing-javascript.md

```diff
@@ -0,0 +1,68 @@
+# Optimizing JavaScript
```

Comment:
I'd mention here tools that help identifying too big packages like bundle analyzer

Reply:
Fixed